### PR TITLE
Reduce finder panel indentation

### PR DESCRIPTION
**Pull request by AmeeMorris**
### 👀 Overview
Pretty small one, just reducing the padding and indentation for the finder panel items.
### 📹 GIF (optional)
Screen_Shot_2019-07-09_at_12_01_26_and_Screen_Shot_2019-07-09_at_12_01_19
### ✅ Checklist
- [ ] Tests written
- [ ] Browser tested
- [ ] Added to documentation
- [ ] Added to storybook

### CODE REVIEW

File: lib/FinderNavigation/styles.scss [hunk 1]
```diff
@@ -26,12 +26,13 @@ $finder-indent-level-support: 10 !default;
...
 
 .finder__item-content {
+  padding-right: $layout-spacing-base/2;
   display: flex;
   cursor: default;
   height: 35px;
   line-height: 35px;
   transition: height 200ms;
-
+  padding-left: $layout-spacing-base/2;
```
Comment (DavidHollins6) [inline lib/FinderNavigation/styles.scss]: One tiny thing, maybe we could do:

`padding: 0 $layout-spacing-base/2;`

**Comment by AmeeMorris** [comment]
[Reduce indentation on finder panel](https://trello.com/c/G89w1N6G/367-reduce-indentation-on-finder-panel)

File: lib/FinderNavigation/styles.scss (hunk 1 above)
Comment (AmeeMorris) [reply lib/FinderNavigation/styles.scss]: I was about to say "no! I only want it on the left!", then I realised at the start of this block I have the exact same thing for padding right. So yes absolutely, nice catch 😅

**Comment by altaudio** [comment]
Nice graphics!

**Pull request merged**